modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -518,6 +533,16 @@ private void updateAssignmentInternal(ConfigurationNotificationEvent<byte[]> ass
                                 ),
                                 groupOptions
                         );
+
+                        replicaMgr.startReplica(grpId,
```

Comment:
Please also rename 
add method shouldHaveReplicationGroupLocally to ReplicaManager and use it instead 
raftMgr.shouldHaveRaftGroupLocally within TableManager.